### PR TITLE
Restructure of the project

### PR DESCRIPTION
**Pull request by Lonrot**
Summarized commit messages:

- Changed project structure to match business logic, not HTTP request.
- new build, target folder deleted and rebuilt.
- Deleted schema injection. Using JPA schema creation.
- Data.sql changed to match the new schema and added more Listings.

### CODE REVIEW

**Review by bm298 — APPROVED** [review]

**Comment by Lonrot** [comment]
@mmoazam @s-kettle  Hi, if you guys have a second, can review this merge , please :smile:

File: src/main/java/springsamurais/toyswapbackend/controller/ListingController.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package springsamurais.toyswapbackend.controller;
+
+import org.springframework.web.bind.annotation.*;
+import springsamurais.toyswapbackend.model.Listing;
+
+
+@RestController
+@RequestMapping
```
Comment (s-kettle) [inline src/main/java/springsamurais/toyswapbackend/controller/ListingController.java:8]: Just checking that adding routing is in the next step? i.e. `RequestMapping("api/v1/")` is planned in the next phase? If so, that's fine.

File: src/test/java/springsamurais/toyswapbackend/repository/ListingRepositoryTest.java [hunk 2]
```diff
@@ -2,6 +2,6 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-class PostInterfaceTest {
+class ListingRepositoryTest {
```
Comment (s-kettle) [inline src/test/java/springsamurais/toyswapbackend/repository/ListingRepositoryTest.java:6]: I don't think repository tests are necessary, might be good with time but these aren't critical.

**Review by s-kettle — APPROVED** [review]
Great stuff, makes more sense how the controllers and service layer is implemented now. Just a couple of comments that aren't major.

**Pull request merged**